polaris-ui/public/index.html

```diff
@@ -19,6 +19,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <script type="module" referrerpolicy="no-referrer-when-downgrade" src="%REACT_APP_CPS_GLOBAL_HEADER_URL%"></script>
```

Comment:
As this stands, the global header code will always be pulled in at runtime even when the feature flag is switched off.  If you wanted to prevent this then `%REACT_APP_CPS_GLOBAL_HEADER_URL%` could become `%REACT_APP_CPS_GLOBAL_HEADER_SCRIPT_TAG%` and in terraform and the UI `.env*` files this value would contain the whole script tag e.g. cps_global_components_script_tag='<script type="module" referrerpolicy="no-referrer-when-downgrade" src="https://sacpsglobalcomponents.blob.core.windows.net/dev/cps-global-components.js"></script>'